ui/App/ProjectScreen/Source/AcceptPatchButton.svelte

```diff
@@ -72,7 +61,10 @@
       To merge this Patch, run this in your terminal:
     </p>
     <Copyable name="command" bind:this={copyable}>
-      <p class="typo-text-small-mono instructions">{instructions}</p>
+      <div class="typo-text-small-mono command-line">
+        upstream patch fetch {patchHandle}<br />
+        git merge {Patch.TAG_PREFIX}{patchHandle}
```

Comment:
Should we also add `git push rad` at the end of the commands? This way the patch actually gets published.
Otherwise new users wouldn't know that this step is required to show this patch as closed in the UI.

Reply:
c3f8fd17723a6c25fa3f05e9bf3b9ee9a24a5185